pkg/agent/secondarynetwork/ipam/ipam_delegator.go

```diff
@@ -0,0 +1,104 @@
+// Copyright 2021 Antrea Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ipam
+
+import (
+	"context"
+	"os"
+	"path/filepath"
+
+	"github.com/containernetworking/cni/pkg/invoke"
+	"github.com/containernetworking/cni/pkg/types"
+	"github.com/containernetworking/cni/pkg/types/current"
+	"k8s.io/klog/v2"
+)
+
+const (
+	ipamGlobal     = "whereabouts"
```

Comment:
Could we remove "Global" from (variable, func) names? It will be confusing, as we will support IPAM in Antrea too (see here: https://github.com/antrea-io/antrea/pull/2740).

If you think that is better, we can call it ipamWhereabouts.

Reply:
@annakhm @gran-vmv